<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>400</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY궁서B</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>회원 정보 조회 프로그램</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>회원 아이디 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>정보 조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>정보 수정</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memberid_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="age_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>이름 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>연락처 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>주소지 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>나이 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
